### ISSUE TITLE
Process XLSX files that have absolute path Target="" for a worksheet

### ISSUE THREAD

**Issue by reedmaniac** [PHPOffice/PhpSpreadsheet#907]
This is:

```
- [x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
A file created by openpyxl will be processed correctly by the Xlsx reader.

### What is the current behavior?
During processing an error is thrown, ex: ```file_get_contents(zip:///private/var/tmp/CXuOzZldUHfRDk6q#xl//xl/worksheets/sheet1.xml): failed to open stream: operation failed```

As you can see, the 'xl/' is duplicated. This path is created [on this line](https://github.com/PHPOffice/PhpSpreadsheet/blob/master/src/PhpSpreadsheet/Reader/Xlsx.php#L218).  Opening up the XSL file in BBEdit allows me to see the XML files.  What seems to be the problem is that the worksheet.xml.rels file has the Target="" attribute as an absolute path:

```<Relationship Id="rId1" Target="/xl/worksheets/sheet1.xml" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/worksheet"/>```

When I open the problematic XSLX file in Numbers and then export as a new XSL, the Target attribute is given a relative path and that new file will work correctly.

### What are the steps to reproduce?
See above.  I have been able to fix the issue by changing the code in Xlsx.php, line 215 to be this, but I am seeking guidance on whether this is a valid solution or if the openpyxl developers should be contacted.  It seems it is not an [uncommon problem though](https://github.com/tidyverse/readxl/issues/294).

```
$path = (substr($fileWorksheet, 0, 1) == '/') ? ltrim($fileWorksheet, '/') : "$dir/$fileWorksheet";

$xml = new XMLReader();
$xml->xml(
    $this->securityScanner->scanFile(
        'zip://' . File::realpath($pFilename) . '#' . $path
    ),
    null,
    Settings::getLibXmlLoaderOptions()
);
```

### Which versions of PhpSpreadsheet and PHP are affected?
PhpSpreadsheet 1.6.0, PHP 7.1

**Comment by tiagof** [comment]
Hi,

having the same issue. I believe it is not fixed yet, right?
I'll try to make a PR to support this.

**Comment by oleibman** [comment]
Fixed by PR #2856 in 2022. Leaving issue closed, removing stale label.
